src/Client/Web/Bit.Client.Web.BlazorUI/Components/FileUploads/BitFileUpload.razor.cs

```diff
@@ -38,22 +38,22 @@ public partial class BitFileUpload : IDisposable
         public long UpLoadedSize { get; set; }
 
         /// <summary>
-        /// General upload status
+        /// General upload status.
         /// </summary>
         public UploadStatus UploadStatus { get; set; }
 
         /// <summary>
-        /// Upload is done in the form of Chunks and this property shows the amount of upload in each Chunk
+        /// Upload is done in the form of chunks and this property shows the progress of upload in each chunk.
         /// </summary>
         public static long ChunkSize => FileSizeHumanizer.OneMegaByte * 10;
 
         /// <summary>
-        /// All selected files
+        /// All selected files.
         /// </summary>
         public IReadOnlyList<BitFileInfo>? Files { get; private set; }
 
         /// <summary>
-        /// Refrence to this component for js accessibility
+        /// Reference to this component for js accessibility.
         /// </summary>
         private DotNetObjectReference<BitFileUpload>? dotnetObjectReference;
```

Comment:
the private members of a type should not have a documentation comment.
please remove it here.